src/api-docs/source/includes/_wallet_rpc_auctions.md

```diff
@@ -1334,6 +1341,998 @@ address | Required | address submitting the bid
+  "hash": "eca1bae09d78312f1f3177eafcde9f48c8d933f807cc276a9224003aba922018",
+  "witnessHash": "f883350cefade0671380fdce71b88996d2a49be30bfd7e3ebb02020e097efe51",
+  "mtime": 1537224055,
+  "version": 0,
+  "inputs": [
+    {
+      "prevout": {
+        "hash": "76ad90abcde417d41d017b8d15b6980a804405aff72537c5cb99eb61a60cfee0",
+        "index": 0
+      },
+      "witness": [
+        "c5af2d7cee71f3079ec24d4e20387388ec44f6617b068e86a495f0b5649f99e916618d25be6d04a038071de074429bd4dbeda021916fc98e4a5a7c0a4e03ca2801",
+        "0329d8c1750b74815e568f2c57ac5492a8e93b3b919334025d37c239b466988590"
+      ],
+      "sequence": 4294967295,
+      "address": "rs1q79u92rxyaxejj3rqyu0kxhzaydxk4ruendeu42"
+    },
+    {
+      "prevout": {
+        "hash": "00c61039848c77220ecafdef2e189c30093ae086c5ed5fce473bd1ec0d0f37fd",
+        "index": 0
+      },
+      "witness": [
+        "264a2153120ae936dd1dd75a0d2174089158b99e77f10db34253ed87216a76de13baa857f0ae5e1c77ac85f0402d74cd3d3154cfafe06e59e07b15c6958c5b4101",
+        "020eb07bca66b10617ffb17fe298a104a21789f4990cedee84577f66fe69656458"
+      ],
+      "sequence": 4294967295,
+      "address": "rs1q0n93g7979gflgq680daemzx8slfg0tqreasrnf"
+    }
+  ],
+  "outputs": [
+    {
+      "value": 2000000,
+      "address": "rs1q79u92rxyaxejj3rqyu0kxhzaydxk4ruendeu42",
+      "covenant": {
+        "type": 5,
+        "action": "REDEEM",
+        "items": [
+          "a761e2b31b2a10714810b3cb439b1ffe347a1019af1932db7e8ac4c62a34224a",
+          "0c000000"
+        ]
+      }
+    },
+    {
+      "value": 1000000720,
+      "address": "rs1qx7yfyscwrwjw86spx88wdhgnj7ljkrxjmljx6j",
+      "covenant": {
+        "type": 0,
+        "action": "NONE",
+        "items": []
+      }
+    }
+  ],
+  "locktime": 0,
+  "hex": "000000000276ad90abcde417d41d017b8d15b6980a804405aff72537c5cb99eb61a60cfee000000000ffffffff00c61039848c77220ecafdef2e189c30093ae086c5ed5fce473bd1ec0d0f37fd00000000ffffffff0280841e00000000000014f178550cc4e9b3294460271f635c5d234d6a8f99050220a761e2b31b2a10714810b3cb439b1ffe347a1019af1932db7e8ac4c62a34224a040c000000d0cc9a3b000000000014378892430e1ba4e3ea0131cee6dd1397bf2b0cd20000000000000241c5af2d7cee71f3079ec24d4e20387388ec44f6617b068e86a495f0b5649f99e916618d25be6d04a038071de074429bd4dbeda021916fc98e4a5a7c0a4e03ca2801210329d8c1750b74815e568f2c57ac5492a8e93b3b919334025d37c239b4669885900241264a2153120ae936dd1dd75a0d2174089158b99e77f10db34253ed87216a76de13baa857f0ae5e1c77ac85f0402d74cd3d3154cfafe06e59e07b15c6958c5b410121020eb07bca66b10617ffb17fe298a104a21789f4990cedee84577f66fe69656458"
+}
+```
+Create `REDEEM` transaction without signing or broadcasting it.
+
+### Params
+Name | Default |  Description
+--------- | --------- | ---------
+name | Required | name to redeem a losing bid for (`null` for all names)
+
+
+
+## createupdate
+
+```shell--vars
+name='possibility'
+```
+
+```shell--cli
+hsw-cli rpc createupdate $name '{"records": [ {"type": "NS", "ns": "ns1.example.com.", "address": "1.2.3.4"} ]}'
```

Comment:
NS records dont have an `address` field anymore, this could be a GLUE4 perhaps?

Reply:
removed address field from the example